person.rb

```diff
@@ -1,8 +1,11 @@
+require_relative 'nameable'
+
 class Person
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, name: 'Unkown', parent_permission: true)
+    super
     @age = age
     @name = name
     @parent_permission = parent_permission
```

Comment:
- [ ] Please try to leverage your person class to take the required arguments as the project guidelines state to make your class pass tests.

errors while testing :
![image](https://user-images.githubusercontent.com/90524466/209227250-23eda3b5-0c12-4314-91fc-b8307a1685c3.png)